src/table/utils/__tests__/handle-scroll.spec.js

```diff
@@ -20,14 +20,14 @@ describe('handle-scroll', () => {
       isRTL = true;
     });
 
-    it('should run preventDefault when scroll to leftmost place in ltr direction', () => {
+    it('should run preventDefault when scroll to the leftmost place and keep scrolling left in LTR direction', () => {
```

Comment:
This is actually what happens right? Since you only scroll once
```suggestion
    it('should run preventDefault when you are at leftmost place and keep scroll left in LTR direction', () => {
```